### PR TITLE
Unflag the static build

### PR DESCRIPTION
**Pull request by matthewp**
## Changes

- What does this change?
- Be short and concise. Bullet points can help!
- Before/after screenshots can be helpful as well.

## Testing

- About 1/2 of the tests are changed to `legacyBuild: true`. Most seem small to fix/change.

## Docs

No docs needed.

### CODE REVIEW

File: packages/astro/src/@types/astro.ts [hunk 1]
```diff
@@ -129,12 +133,19 @@ export interface AstroUserConfig {
 		 */
 		drafts?: boolean;
 		/**
-		 * Experimental: Enables "static build mode" for faster builds.
+		 * Enables "legacy build mode" for compatibility with older Astro versions.
 		 * Default: false
 		 */
+		legacyBuild?: boolean;
```
Comment (jonathantneal) [inline packages/astro/src/@types/astro.ts:139]: 👍

File: yarn.lock [hunk 2]
```diff
@@ -7095,7 +7095,7 @@ sander@^0.5.0:
     mkdirp "^0.5.1"
     rimraf "^2.5.2"
 
-sass@^1.49.0:
+sass@^1.49.0, sass@^1.49.8:
```
Comment (jonathantneal) [inline yarn.lock:7098]: ❓

**Review by jonathantneal — APPROVED** [review]
Left a note about a change to `yarn.lock`, but otherwise nice!

**Review by tony-sull — APPROVED** [review]
Looks good to me too (pending Jonathan's `yarn.lock` question)

File: packages/astro/src/@types/astro.ts [hunk 3]
```diff
@@ -27,8 +27,12 @@ export interface CLIFlags {
 	hostname?: string;
 	port?: number;
 	config?: string;
+	/**
+	 * @deprecated
```
Comment (FredKSchott) [inline packages/astro/src/@types/astro.ts:31]: nit: `/** @deprecated */`

**Pull request merged**